test/cypress/integration/simplified-account/service-settings/webhooks/webhook-event.cy.js

```diff
@@ -106,6 +106,11 @@ describe('for an admin', () => {
     cy.get('h1.govuk-heading-l').should('have.text', 'Payment captured')
   })
 
+  it('should show event body in detail component', () => {
+    cy.get('.govuk-details summary').should('contain.text', 'Payment captured event body')
+    cy.get('.govuk-details div.govuk-details__text pre code').should('contain.text', '{')
```

Comment:
does the formatting of the JSON make it awkward to test for the full event body?

Reply:
I've had another look at this. I've got a matching object to compare. It's just matching in the string values, so I'm not sure how reliable that is going to be. Do you know if there's a straight forward way for the cypress test to match these as json objects?

Reply:
I think matching the stringified JSON works, seems like any option may be a tad awkward. I couldn't find any sensible alternative in the cypress docs